src/apps/components/AppAvatar/AppAvatar.tsx

```diff
@@ -28,8 +28,8 @@ export const AppAvatar: React.FC<{
       borderRadius={2}
       borderWidth={1}
       borderColor="default1"
-      borderStyle={"solid"}
+      borderStyle="solid"
     >
-      <GenericAppIcon size="large" color="default2" />
+      <GenericAppIcon size="medium" color="default2" />
```

Comment:
This component is used in other contexts, like AppDetails / AppView, is this breaking UI?

Reply:
I've just double checked and it didn't break UI in list and details of an app. Only icon inside is a bit smaller:

| Before | After |
| ------- | ------- |
| ![CleanShot 2024-01-30 at 12 37 52@2x](https://github.com/saleor/saleor-dashboard/assets/9116238/8fd9546b-ab86-4494-9a37-049792d7e66e) | ![CleanShot 2024-01-30 at 12 37 45@2x](https://github.com/saleor/saleor-dashboard/assets/9116238/0dfd95f3-eaf6-42d4-be22-8c529d3844a6) |